.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: go
 go:
-- 1.10rc1
+- tip
```

Comment:
is tip pointing to latest release or is the same thing as pointing to master ?

Reply:
Not sure. Changing to master just in case